### DISCUSSION TITLE
Do all the translation files get loaded to the frontend?

### DISCUSSION THREAD

**Discussion by juniorforlife** [sergiodxa/remix-i18next#160] [Q&A] [answered]
I have 22 translation files present within the `public/locales` directory as per the documentation. 
I'm concerned about the potential impact on the application size. Will all these files be shipped to the browser, thereby significantly increasing the size of my application? If so then how do I only load the one file being used?

**Comment by sergiodxa** [comment, marked as answer]
Usually i18next backends load only the translations you need, unless you explicitly ask it to load all translation files, but it will depend on your i18next configuration and backend, remix-i18next doesn't load your translation files.
